tests/test_scylla_repository.py

```diff
@@ -32,18 +34,28 @@ class TestScyllaRepository:
         ("release:4.2", '4.2'),
         ("release:4.1", '4.1'),
         ("release:4.0", '4.0'),
+        ("release:6.1", '6.1'),
+        ("release:6.1:debug", '6.1'),
     ])
     def test_setup_release_oss(self, version, expected_version_string):
         cdir, version = scylla_setup(version=version, verbose=True)
         assert expected_version_string in version
 
-    def test_setup_unstable_master_new_url(self):
-        cdir, version = scylla_setup(version="unstable/master:2023-09-05T13:59:07Z", verbose=True)
-        assert version == '5.4.0-dev'
+    @pytest.mark.parametrize(argnames=['version', 'expected_version_string'], argvalues=[
+        ("unstable/master:2023-09-05T13:59:07Z", '5.4.0-dev'),
+        ("unstable/master:2023-09-05T13:59:07Z:debug", '5.4.0-dev'),
```

Comment:
so how do you know if it actually took a debug version?

Reply:
You dont, there is an another test that tests if the actual debug version was taken